src/mpi/romio/adio/ad_gpfs/ad_gpfs_wrcoll.c

```diff
@@ -360,7 +360,7 @@ void ADIOI_GPFS_WriteStridedColl(ADIO_File fd, const void *buf, int count,
             ADIOI_OneSidedWriteAggregation(fd, offset_list, len_list, contig_access_count, buf,
                                            datatype, error_code, firstFileOffset, lastFileOffset,
                                            currentValidDataIndex, fd_start, fd_end, &holeFound,
-                                           noStripeParms);
+                                           &noStripeParms);
```

Comment:
There are two things going on in this commit (`making romio GPFS able to build and run again`). Can you split them?

Reply:
Ideally this commit should also move to separate PR as it is not related to the Alltoallv fix

Reply:
Is that really better?  It turns the main PR into something that can't pass CI until a separate PR is checked in.  This way you at least have a PR that can be built and run without intra-PR dependencies.  I can change it, just you won't be able to test this PR or have it pass CI in the meantime

The fixes related to different build environments I agree could be easily moved to a different PR

Reply:
I separated it to different PRs

Reply:
Thanks. When we review or cherry-pick, we always do it on a per-PR basis. So it is desirable to make each PR with its obvious scope. It is not the number of PRs that inhibit reviewing. It is a finite amount of changes that we need to review one way or another. Having them separate allows us to review in a simpler context thus help making the process easier and faster.